firmware/src/dimmer_model.c

```diff
@@ -1315,7 +1315,7 @@ static const PIDDescriptor SUBDEVICE_PID_DESCRIPTORS[] = {
   {PID_CLEAR_STATUS_ID, (PIDCommandHandler) NULL, 0u,
     DimmerModel_ClearStatusId},
   {PID_SUB_DEVICE_STATUS_REPORT_THRESHOLD,
-    DimmerModel_GetSubDeviceReportingThreshold, 1u,
```

Comment:
How did you spot this, it looks like we don't have any tests for it in the RDM tests?